bin/setup.js

```diff
@@ -12,31 +12,40 @@ var saveRequested = process.env.npm_config_save
 // This block only executes if Hoodie is installed with `-S` or `--save` flags.
 if (saveRequested) {
   var pathToAppRoot = path.resolve('..', '..')
-  var packageJson = require(path.join(pathToAppRoot, 'package.json'))
 
-  packageJson.scripts = packageJson.scripts || {}
+  try {
+    var packageJson = require(path.join(pathToAppRoot, 'package.json'))
 
-  if (packageJson.scripts[ 'start' ]) {
-    if (!(packageJson.scripts[ 'start' ].startsWith('hoodie'))) {
-      log.info('setup', 'start script already set to "' + packageJson.scripts[ 'start' ] +
-      ', you can start hoodie with "npm run start-hoodie" instead')
-      packageJson.scripts[ 'start-hoodie' ] = 'hoodie'
+    packageJson.scripts = packageJson.scripts || {}
+
+    if (packageJson.scripts[ 'start' ]) {
+      if (!(packageJson.scripts[ 'start' ].startsWith('hoodie'))) {
+        log.info('setup', 'start script already set to "' + packageJson.scripts[ 'start' ] +
+        ', you can start hoodie with "npm run start-hoodie" instead')
+        packageJson.scripts[ 'start-hoodie' ] = 'hoodie'
+      }
+    } else {
+      packageJson.scripts[ 'start' ] = 'hoodie'
     }
-  } else {
-    packageJson.scripts[ 'start' ] = 'hoodie'
-  }
 
-  var newPackageJson = JSON.stringify(packageJson, null, 2)
+    var newPackageJson = JSON.stringify(packageJson, null, 2)
 
-  fs.writeFile(path.join(pathToAppRoot, 'package.json'), newPackageJson,
-    function (error) {
-      if (error) {
-        log.error('setup', 'Could not create package.json at ' + path.join(pathToAppRoot, 'package.json'))
-        log.error(error)
-        return
-      }
-      log.info('setup', 'Start your Hoodie app with "npm start"')
-    })
+    fs.writeFile(path.join(pathToAppRoot, 'package.json'), newPackageJson,
+      function (error) {
+        if (error) {
+          log.error('setup', 'Could not create package.json at ' + path.join(pathToAppRoot, 'package.json'))
+          log.error(error)
+          return
+        }
+        log.info('setup', 'Start your Hoodie app with "npm start"')
+      })
+  } catch (error) {
+    if (error.code !== 'MODULE_NOT_FOUND') {
+      throw error
+    }
+    log.error('setup', 'Could not find package.json at ' + path.join(pathToAppRoot, 'package.json'))
+    log.error(error)
```

Comment:
Yeah, a warning should be sufficient. Not really familiar with the logging system.